French/panique/frapper-le-mur.md

```diff
@@ -1,3 +1,4 @@
 Alors que vous laissez la panique s'emparer de vous, vous commencez à courir dans tous les sens.
 
-Finalement, emporté par votre élan, vous finissez pas frapper un mur et vous écrouler à terre. Bien joué !
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.